### PR TITLE
Update Lemonade and Credijusto case studies

### PR DESCRIPTION
**Pull request by zchase**
Fixes https://github.com/pulumi/marketing/issues/35
Fixes https://github.com/pulumi/marketing/issues/37

### CODE REVIEW

File: themes/default/content/case-studies/credijusto.md [hunk 1]
```diff
@@ -105,6 +105,19 @@ Pulumi made network configurations extremely simple and easy to replicate across
...
+each application to scale with demand.
+
+> "Our database setup relies on a dedicated Amazon Aurora cluster for each product. Everything
+> is configured by Pulumi, by a custom resource that we built to create and manage all related
+> resources (credentials, Option Groups, Parameter Groups, Subnets, Security Groups, Policies, etc.).
+> This ensures a standard configuration across all of our databases."
+>
+> &mdash; Fernando Carlietti, Lead DevOps Engineer, Credijusto.
+
 <img class="block mx-auto md:max-w-4xl my-8" src="/images/case-studies/credijusto-architecture.png" alt="Credijusto architecture">
```
Comment (isaac-pulumi) [inline themes/default/content/case-studies/credijusto.md:121]: @zchase can you replace with this image (attached)?
Credijusto - Architecture

**Review by isaac-pulumi — COMMENTED** [review]
text looks good - please swap the architecture diagram.

File: themes/default/content/case-studies/credijusto.md (hunk 1 above)
Comment (zchase) [reply themes/default/content/case-studies/credijusto.md:121]: 👍 Yeah not sure why it got lost when I updated it the first time.

**Review by isaac-pulumi — APPROVED** [review]
Looks great!

**Pull request merged**